source/includes/_investors.md

```diff
@@ -552,7 +552,7 @@ Updates the investor's classification.
 
 ```http
 
-GET /platformApi/investors/{clientId}/kyc HTTP/1.1
+GET /platformApi/investors/GOJI-1234/kyc HTTP/1.1
```

Comment:
Looks like the `{paramName}` pattern is used a lot elsewhere, e.g.

* https://docs.api.goji.investments/#payments-investors-post-platformapi-investors-clientid-accounts-accounttype-withdrawal
* https://docs.api.goji.investments/#bond-management-post-platformapi-investors-clientid-accounts-accounttype-investments

Worth sticking with it?  Or are we moving towards the example approach instead?  Or did I just start doing this inadvertently in mine and set a bad example?

Reply:
I went with your example which I personally think is better that we show real examples.

Reply:
I can't see anywhere that explains what `GOJI-1234` represents though (i.e. this is the investor client id)